### ISSUE TITLE
"Inverse hashtags" for grouping toots that aren't visible by default, but are public

### ISSUE THREAD

**Issue by rtucker** [tootsuite/mastodon#2657]
G'day,

There are a number of discussion-types that are dominating the timelines and making it extremely difficult to discuss anything else.  An example of this is administrative/meta stuff -- most of my federated timeline is now arguments and debates between admins, which is not a great experience for our members.

The current solution for the admin meta-discussion is to use a mailing list, use Matrix, use Discourse, use IRC, or use some other medium that is not Mastodon.  This is undesirable, as we already have a chat system that every Mastodon admin uses: Mastodon.  Keeping this discussion open but not visible by default is also extremely important, and siloing it off-network is not healthy.

In short, we need a way to channelize discussions into their own "broadcast domain" to reverse the trend towards becoming a network for discussion of the network itself.

* * * *

- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [X] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).
